src/shared/state/statebase.cpp

```diff
@@ -30,35 +30,35 @@ QJsonValue::Type AddonStateBase::typeToQJsonValueType(QString type) {
 }
 
 // static
-StateHash AddonStateBase::parseManifest(const QJsonObject& manifest) {
+StateHash StateBase::parseAddonManifest(const QJsonObject& initialState) {
```

Comment:
well, this is wrong, right? Why having a "parseAddonManifest" if addons are not shared?

Maybe move it in Addon.

Reply:
Agree on moving it to addons.

Reply:
Ok, so moving this to addons would require the validation code to move to addons as well and I did not like that. I instead just changed the wording on the doc comments and function names to not refer to addons specifically. Let me know what you think.